verticapy/tests/vDataFrame/test_vDF_combine_join_sort.py

```diff
@@ -198,6 +198,25 @@ def test_vDF_join(self, market_vd):
         # SELECT COUNT(*) FROM cross_join WHERE Name2 IS NULL;
         assert cross_join["Name2"].count() == 63616
 
+        # join directly with a Vertica table
+        not_dried._VERTICAPY_VARIABLES_["cursor"].execute(
```

Comment:
Could we use vDataFrame.to_db() instead?

Reply:
We could add a method for "local temporary tables in 'to_db', it can be a good idea.

Reply:
That might unnecessary for now.